changelog.md

```diff
@@ -6,6 +6,10 @@
 - react-native-bpk-component-card:
   - New component BpkCard, see http://backpack.prod.aws.skyscnr.com/components/native/cards/
 
+**Fixed:**
+- bpk-component-image:
+  - Loading spinner will now be removed from DOM once image is displayed
```

Comment:
You've also removed the image mixins from `bpk-mixins` which is a breaking change. I know we have previously talked about the fact that we know nobody is using them means we could get away with just removing them - however we also recently agreed to stick to strict semver at all times :P

Reply:
Sticking to semver doesn't mean we can't break stuff though? It just means we have to increment the major version when we do!

Reply:
Oh I see

Reply:
@shaundon , @georgegillams the heading for `bpk-component-image` is fine, it's the missing `bpk-tokens` breaking change that i am on about :).

As a general rule of thumb - we want to minimise breaking changes as much as possible. This one is justified tho because those api's are confusing and should not have existed in the first place.

Reply:
Have added a `breaking` entry for `bpk-mixins`